--- a/gfx/common/wayland_common.c
+++ b/gfx/common/wayland_common.c
@@ -1097,6 +1097,7 @@
    /* Bind SHM based wl_buffer to wl_surface until the vulkan surface is ready.
     * This shows the window which assigns us a display (wl_output)
     * which is useful for HiDPI and auto selecting a display for fullscreen. */
+#if 0
    if (video_monitor_index == 0 && wl_list_length (&wl->all_outputs) > 1)
    {
       if (!wl_draw_splash_screen(wl))
@@ -1109,6 +1110,7 @@
       wl_display_dispatch(wl->input.dpy);
    }
 #endif
+#endif
 
    /* Ignore configure events until splash screen has been replaced */
    wl->ignore_configuration = true;
