--- a/cheevos/cheevos.c
+++ b/cheevos/cheevos.c
@@ -454,6 +454,13 @@
       }
    }
 
+   const char* cheevos_cmd = settings->arrays.cheevos_cmd;
+   if (cheevos_cmd[0]) {
+     char cmdargs[256];
+     snprintf(cmdargs, sizeof(cmdargs), "%s %u \"%s\" \"%s\" > /dev/null 2>/dev/null", cheevos_cmd, cheevo->id, cheevo->title, cheevo->description);
+     system(cmdargs);
+   }
+
 #ifdef HAVE_AUDIOMIXER
    /* Play the unlock sound */
    if (settings->bools.cheevos_unlock_sound_enable)
--- a/configuration.c
+++ b/configuration.c
@@ -1621,6 +1621,7 @@
 
 #ifdef HAVE_CHEEVOS
    SETTING_ARRAY("cheevos_custom_host",          settings->arrays.cheevos_custom_host, false, NULL, true);
+   SETTING_ARRAY("cheevos_cmd",                  settings->arrays.cheevos_cmd, false, NULL, true);
    SETTING_ARRAY("cheevos_username",             settings->arrays.cheevos_username, false, NULL, true);
    SETTING_ARRAY("cheevos_unlock_sound",         settings->arrays.cheevos_unlock_sound, false, NULL, true);
    SETTING_ARRAY("cheevos_password",             settings->arrays.cheevos_password, false, NULL, true);
--- a/configuration.h
+++ b/configuration.h
@@ -538,6 +538,7 @@
       char cheevos_unlock_sound[128];
       char cheevos_token[32];
       char cheevos_leaderboards_enable[32];
+      char cheevos_cmd[256];
       char video_context_driver[32];
       char audio_driver[32];
       char audio_resampler[32];
