diff --git a/src/frontend/mame/mameopts.cpp b/src/frontend/mame/mameopts.cpp
index 478bf7b4117..901be88e763 100644
--- a/src/frontend/mame/mameopts.cpp
+++ b/src/frontend/mame/mameopts.cpp
@@ -56,10 +56,13 @@ void mame_options::parse_standard_inis(emu_options &options, std::ostream &error
 	}
 
 	// now parse the main INI file following the potentially updated search path
 	parse_one_ini(options, emulator_info::get_configname(), OPTION_PRIORITY_MAME_INI, &error_stream);
 
+	// parse Batocera-specific configuration
+	parse_one_ini(options, "batocera", OPTION_PRIORITY_MAXIMUM, &error_stream);
+
 	// debug mode: parse "debug.ini" as well
 	if (options.debug())
 		parse_one_ini(options, "debug", OPTION_PRIORITY_DEBUG_INI, &error_stream);
 
 	// if we have a valid system driver, parse system-specific INI files
