diff --git a/Config.in b/Config.in
--- a/Config.in
+++ b/Config.in
@@ -2,6 +2,7 @@
 
 menu "System"
   source "$BR2_EXTERNAL_BATOCERA_PATH/package/batocera/core/batocera-system/Config.in"
+  source "$BR2_EXTERNAL_BATOCERA_PATH/package/batocera/focused-system/manufacturer-systems/Config.in"
   source "$BR2_EXTERNAL_BATOCERA_PATH/package/batocera/core/batocera-desktopapps/Config.in"
   source "$BR2_EXTERNAL_BATOCERA_PATH/package/batocera/core/batocera-splash/Config.in"
   source "$BR2_EXTERNAL_BATOCERA_PATH/package/batocera/core/batocera-scripts/Config.in"
