diff --git a/package/batocera/boot/batocera-shim-signed-efi-helpers-ia32/batocera-shim-signed-efi-helpers-ia32.hash b/package/batocera/boot/batocera-shim-signed-efi-helpers-ia32/batocera-shim-signed-efi-helpers-ia32.hash
index ced09c71dc..7ddd76866e 100644
--- a/package/batocera/boot/batocera-shim-signed-efi-helpers-ia32/batocera-shim-signed-efi-helpers-ia32.hash
+++ b/package/batocera/boot/batocera-shim-signed-efi-helpers-ia32/batocera-shim-signed-efi-helpers-ia32.hash
@@ -1,2 +1,2 @@
 # From https://packages.debian.org/bookworm/i386/shim-helpers-i386-signed/download :
-sha256	f7f20896b122cf7bbb19804a07b1cb5eec4aa099be51c778a51d2cb3c08511a4  shim-helpers-i386-signed_1+15.8+1~deb12u1_i386.deb
+sha256	6b357e11e55871bd27d7f55d8c0384039597fa033b8ddd32d432129391d1d5b7  shim-helpers-i386-signed_1+16.1+2~deb12u1_i386.deb
diff --git a/package/batocera/boot/batocera-shim-signed-efi-helpers-ia32/batocera-shim-signed-efi-helpers-ia32.mk b/package/batocera/boot/batocera-shim-signed-efi-helpers-ia32/batocera-shim-signed-efi-helpers-ia32.mk
index 2ed85060c9..d979753b29 100644
--- a/package/batocera/boot/batocera-shim-signed-efi-helpers-ia32/batocera-shim-signed-efi-helpers-ia32.mk
+++ b/package/batocera/boot/batocera-shim-signed-efi-helpers-ia32/batocera-shim-signed-efi-helpers-ia32.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-BATOCERA_SHIM_SIGNED_EFI_HELPERS_IA32_VERSION = 1+15.8+1~deb12u1
+BATOCERA_SHIM_SIGNED_EFI_HELPERS_IA32_VERSION = 1+16.1+2~deb12u1
 BATOCERA_SHIM_SIGNED_EFI_HELPERS_IA32_SITE = https://ftp.debian.org/debian/pool/main/s/shim-helpers-i386-signed
 BATOCERA_SHIM_SIGNED_EFI_HELPERS_IA32_SOURCE = shim-helpers-i386-signed_$(BATOCERA_SHIM_SIGNED_EFI_HELPERS_IA32_VERSION)_i386.deb
 BATOCERA_SHIM_SIGNED_EFI_HELPERS_IA32_INSTALL_IMAGES = YES
diff --git a/package/batocera/boot/batocera-shim-signed-efi-ia32/batocera-shim-signed-efi-ia32.hash b/package/batocera/boot/batocera-shim-signed-efi-ia32/batocera-shim-signed-efi-ia32.hash
index 583bbc24c2..51602dfe12 100644
--- a/package/batocera/boot/batocera-shim-signed-efi-ia32/batocera-shim-signed-efi-ia32.hash
+++ b/package/batocera/boot/batocera-shim-signed-efi-ia32/batocera-shim-signed-efi-ia32.hash
@@ -1,2 +1,2 @@
 # From https://packages.debian.org/bookworm/i386/shim-signed/download :
-sha256	4ed64ef758a5d9a023bce832f615447ebea7b0f6124c3f3c2832e3f03d08954b  shim-signed_1.44~1+deb12u1+15.8-1~deb12u1_i386.deb
+sha256	c3497d2a2527e04e074c668cf4a524f6c05b6a05b150c8c888547d4e438dbca9  shim-signed_1.51~1+deb12u1+16.1-2~deb12u1_i386.deb
diff --git a/package/batocera/boot/batocera-shim-signed-efi-ia32/batocera-shim-signed-efi-ia32.mk b/package/batocera/boot/batocera-shim-signed-efi-ia32/batocera-shim-signed-efi-ia32.mk
index 4df57ab762..5efaa802ab 100644
--- a/package/batocera/boot/batocera-shim-signed-efi-ia32/batocera-shim-signed-efi-ia32.mk
+++ b/package/batocera/boot/batocera-shim-signed-efi-ia32/batocera-shim-signed-efi-ia32.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-BATOCERA_SHIM_SIGNED_EFI_IA32_VERSION = 1.44~1+deb12u1+15.8-1~deb12u1
+BATOCERA_SHIM_SIGNED_EFI_IA32_VERSION = 1.51~1+deb12u1+16.1-2~deb12u1
 BATOCERA_SHIM_SIGNED_EFI_IA32_SITE = https://ftp.debian.org/debian/pool/main/s/shim-signed
 BATOCERA_SHIM_SIGNED_EFI_IA32_SOURCE = shim-signed_$(BATOCERA_SHIM_SIGNED_EFI_IA32_VERSION)_i386.deb
 BATOCERA_SHIM_SIGNED_EFI_IA32_INSTALL_IMAGES = YES
